.github/workflows/build_test_fmt.yml

```diff
@@ -24,5 +24,28 @@ jobs:
     - name: Build
       run: cargo build
 
-    - name: Run Unit Tests and Documentation examples
+    - name: Build and install gccrs
+      run: |
+        sudo apt-get install -y \
+          automake \
+          autoconf \
+          libtool \
+          autogen \
+          bison \
+          flex \
+          libgmp3-dev \
+          libmpfr-dev \
+          libmpc-dev \
+          build-essential \
+          gcc-multilib \
+          g++-multilib
+        git clone --depth 1 https://github.com/Rust-GCC/gccrs
+        cd gccrs
+        mkdir gccrs-build
+        cd gccrs-build
+        ../configure --enable-languages=rust --disable-bootstrap --enable-multilib
+        make -j$(nproc)
+        sudo make install
```

Comment:
Can gccrs provide prebuilt versions as GHA artifacts? That would save a lot of time on the cargo-gccrs CI.

Reply:
I have no idea on how to set this up but it would be really nice

Reply:
I was just about to suggest this. I think gccrs should provide this for you, lets take a look at what we can do there. We do have a docker image being built off master for every merge.

https://github.community/t/how-do-i-use-custom-docker-image-with-github-actions/18112

So we have a docker image over on: https://hub.docker.com/repository/docker/philberty/gccrs

This would mean in theory things are simplified to install whatever rust components you will need.